examples/index.md

```diff
@@ -162,3 +162,292 @@ Content-Type: application/vnd.api+json
+      "title": "Invalid Attribute",
+      "source": { "pointer": "data/attributes/first-name" },
+      "detail": "First name must contain at least three characters."
+    },
+    {
+      "title": "Invalid Attribute",
+      "source": { "pointer": "data/attributes/first-name" },
+      "detail": "First name must contain an emoji."
+    }
+  ]
+}
+```
+
+If the API docs specified:
+
+> | Error Code | Title                                                                             |
+> |------------|-----------------------------------------------------------------------------------|
+> |  11        | "Invalid Attribute"                                                               |
+> |  123       | "too short"                                                                       |
+> |  124       | "emoji missing"                                                                   |
+> |  225       | "password must contain a letter, number, space, neologism, and special character" |
+> |  226       | "passwords do not match"                                                          |
+> |  227       | "password cannot be one of last five passwords"                                   |
+
+Multiple errors on `password` attribute, with error "code":
+
+```http
+HTTP/1.1 422 Unprocessable Entity
+Content-Type: application/vnd.api+json
+
+{
+  "errors": [
+    {
+      "code":   "123",
+      "source": { "pointer": "data/attributes/first-name" },
+      "detail": "First name must contain at least three characters."
+    },
+    {
+      "code":   "225",
+      "source": { "pointer": "data/attributes/password" },
+      "detail": "Frobnicate is not a neologism."
+    },
+    {
+      "code":   "226",
+      "source": { "pointer": "data/attributes/password" },
+      "detail": "Password and password confirmation do not match."
+    }
+  ]
+}
+```
+
+Multiple errors on `first-name` attribute, with error "code", and "title":
+
+```http
+HTTP/1.1 422 Unprocessable Entity
+Content-Type: application/vnd.api+json
+
+{
+  "jsonapi": { "version": "1.0" },
+  "errors": [
+    {
+      "code":   "123",
+      "title":  "too short",
+      "source": { "pointer": "data/attributes/first-name" },
+      "detail": "First name must contain at least three characters."
+    },
+    {
+      "code":   "124",
+      "title":  "emoji missing",
+      "source": { "pointer": "data/attributes/first-name" },
+      "detail": "First name must contain an emoji"
+    }
+  ]
+}
+```
+
+Notice that this response includes not only the "errors" top-level member,
+but the "jsonapi" top-level member. It may also have include the "meta" or "links"
+top-level members, but not the "data" member.
+
+```http
+PATCH /posts/1
+
+{ "datum": { } }
+```
+
+```http
+HTTP/1.1 422 Bad Request
+Content-Type: application/vnd.api+json
+
+{
+  "errors": [
+    {
+      "title":  "Missing Data Member",
+      "source": { "source": "/data" },
+      "detail": "The required parameter, data, is missing."
```

Comment:
Can you get rid of `detail` here? It basically restates the title, so it isn't needed. More importantly though, I don't want to call `data` a "parameter" since that further blurs the distinction between `source` and `parameter`; thus far, the only thing we've called "parameters" are the URI query params.